src/common/utility.js

```diff
@@ -456,16 +453,19 @@ const handleEURedirect = (country, full_domain) => {
     }
 }
 
+const getSubdomain = () => isBrowser() && window.location.hostname.split('.').slice(0, -2).join('.')
+
+export const isEuDomain = () => !!eu_domains.includes.getSubdomain()
+
 export const handleRedirect = (residence, current_client_country, full_domain) => {
-    const subdomain = window.location.hostname.split('.').slice(0, -2).join('.')
     const country = residence ? residence : current_client_country
 
     if (isTestlink()) {
         return false
-    } else if (eu_domains.some((e) => subdomain.includes(e))) {
+    } else if (eu_domains.some((e) => getSubdomain().includes(e))) {
         handleEURedirect(country, full_domain)
```

Comment:
Do you think we still need this. We are now switching off the redirection in `eu.deriv.com`